build/server-image/kubectl/Dockerfile

```diff
@@ -14,10 +14,16 @@
 
 # Dockerfile used for the kubectl image.
 
-ARG BASEIMAGE
 ARG BINARY
 
 
-FROM "${BASEIMAGE}"
+FROM registry.k8s.io/build-image/debian-base:bookworm-v1.0.0
 COPY ${BINARY} /bin/
+
+# Install additional utilities
+RUN apt update \
```

Comment:
Let's explicitly enumerate all the packages we need? (so if someone gets dropped off a base image in the future we still install it explicitly)

Reply:
Sounds about right. Updated the PR.